indexer/services/socks/src/lib/message-forwarder.ts

```diff
@@ -165,13 +165,13 @@ export class MessageForwarder {
 
     if (!this.subscriptions.subscriptions[message.channel] &&
       !this.subscriptions.batchedSubscriptions[message.channel]) {
-      logger.debug({
-        at: 'message-forwarder#forwardMessage',
-        message: 'No clients to forward to',
-        messageId: message.id,
-        messageChannel: message.channel,
-        contents: message.contents,
-      });
+      // logger.debug({
```

Comment:
Should we just remove the logging altogether or do you want to keep it around for some reason?

Reply:
I think having it around for now is fine